usd_qtpy/editor.py

```diff
@@ -94,5 +102,23 @@ def update_panel_checkstate():
 
         panels_menu.aboutToShow.connect(update_panel_checkstate)
 
+        # Render menu
+
+        render_menu = menubar.addMenu("Render")
+        render_labels = "Playblast" , "Snapshot"
+        render_actions = {label : render_menu.addAction(label) for label in render_labels}
+
+        # Testing 
+        print("\n".join(render_util.getAllRenderEngineNames()))
+        goal_file = R"C:\dump\playblastview##.##.png"
+        
+        def render_snapshot(stage,stageview,path):
+            cam = render_util.playblast.cameraFromView(stage,stageview)
+            render_util.playblast.renderPlayblast(stage,path,"1",1920,cam,renderer="GL")
+
+        render_snap = partial(render_snapshot,self._stage,self._stageview,goal_file)
```

Comment:
PEP8 check your linter; these are minor things though.
```suggestion
        render_snap = partial(render_snapshot, self._stage, self._stageview, goal_file)
```